UnityProject/Assets/Scripts/Objects/Disposals/DisposalVirtualContainer.cs

```diff
@@ -169,8 +169,11 @@ private void ThrowContainedItems(Vector3 throwVector)
 		{
 			foreach (ObjectBehaviour item in containedItems)
 			{
+				if(item == null) continue;
```

Comment:
Any idea why it is null? Despawned during traversal?

Reply:
So it becomes null after being ejected but before thrown? That's odd?

Reply:
> Any idea why it is null? Despawned during traversal?

Yeah thats what im guessing

Was nulling on this line, throw vector wouldnt be null so must be the item

Vector3 vector = item.transform.rotation * throwVector;